Add new full guide Zeus - ES

Darth-Coin opened the pull request:
Add a full new step by step guide about Zeus embedded node and new images.
Later will make a new PR for EN language.

Asi0Flammeus commented:
Thank you for your contribution! 
You have really enhanced the tutorial for Zeus but before merging I need to make sure that file format that were never used before (eg. .mp4 and .gif) are supported by our codebase. 
Will keep you in touch in the following days

Darth-Coin commented:
@Asi0Flammeus sure, that's why I started first with the ES version. If is necessary to remove something let me know so I will not use it in the EN version.

Asi0Flammeus committed:
events, add BEA Germany (#985)

Asi0Flammeus committed:
event, fix address bea germany

Asi0Flammeus committed:
event, add bitcoin alps conference (#986)

Asi0Flammeus committed:
event, fix date

Henry-Da0 committed:
error fixing - eco102 - en (#782)

* Update en.md

* Update en.md

ProfEduStream committed:
BTC302 update (#852)


* Delete courses/btc302/assets/fr/chapter27/43.webp

* Add files via upload

* Update fr.md

* Delete courses/btc302/assets/fr/chapter27/43-bis.jpg

* Add files via upload

* Add files via upload

---------

Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

Asi0Flammeus committed:
btc302, change FR proofreader id

Henry-Da0 committed:
Update en.md (#879)

BitPreto committed:
Fix 50-planb-tags.md (#908)

* Update 50-planb-tags.md

the file still misses one last tag, I hope to update it in the following 2 weeks but as of now I have no more tag to add

* Update 50-planb-tags.md

jinformatique committed:
[RESOURCE] Add This Week in Bitcoin podcast + Fix Robosats typo (#928)

* added new podcast This Week in Bitcoin + added info: bitcoin dad pod is on pause

* more info added

* bitcoin-only typo correction

* changes typo for bitcoi only to bitcoin-only + added some info for 2 podcasts bitcoin dad pod and ThisWeekInBitcoin

Rogzy-DB committed:
remove only 2 images of courses with texte translation (#940)

caeher committed:
doc(cyp201): fix step 2 for determine P, M = 950 (#971)

Asi0Flammeus committed:
professors, fix giacomo profile pic

Henry-Da0 committed:
proofreading phi201 - vi (#804)

* Update vi.md

* Update vi.md

* Update vi.md

BitPreto committed:
missing professors with a few translations (#996)

jramos0 committed:
Bitcoin Bali events (#995)

* Bitcoin for Kids Bitcoin Bali Events, #994

* Bitdevs Bitcoin Bali Events, #994

* learning cli Bitcoin Bali Events, #994

* fix bug in learning cli Bitcoin Bali Events, #994

* fix bug in learning cli Bitcoin Bali Events, #994

* add Bringin Bitcoin Afterpary, #942

LoicPandul committed:
[TUTORIAL] Update repo name in tutos (#992)

* set the correct repo name

Asi0Flammeus committed:
script, update get reward evaluation

Asi0Flammeus committed:
Norwegian bokmål initial upload (#1008)

* Norwegian-Bokmal translation of BTC101

* Rename en_nb-NO.md to nb-NO.md

* add initial batch translation

* add initial batch translation

---------

Co-authored-by: MarJJ <mariannacon@protonmail.com>

Asi0Flammeus committed:
nb-NO, fix yml properties for builders

Asi0Flammeus committed:
nb-NO, fix yml properties for bet

Asi0Flammeus committed:
books, fix yml properties

Asi0Flammeus committed:
resources, replace webp image and ref

Asi0Flammeus committed:
eco102, remove yt links except in fr

Asi0Flammeus committed:
nb-NO, fix uuids

Asi0Flammeus committed:
nb-NO, fix uuids

LoicPandul committed:
PHI201 review (#1003)

* Higher resolution for the blurry image of Von Mises.

* Higher resolution for the blurry image La Liberté guidant le peuple

* Higher resolution for the blurry image of Voltaire

* img-012.webp  -> Smaller image size to reduce blurriness.

* Higher resolution for the blurry image of Périclès

* Update img-001.webp

* Update img-108.webp

* Update img-008.webp

* Update img-011.webp

* X html

* Update img-004.webp

* Update img-041.webp

* Update img-034.webp

* Update img-044.webp

* Update img-046.webp

* Update img-047.webp

* Update img-031.webp

* Update img-032.webp

* Update img-051.webp

* Update img-060.webp

* Update img-078.webp

* Update img-053.webp

* Update img-066.webp

* Update img-081.webp

* Update img-071.webp

* Update img-063.webp

* Update img-076.webp

* Update img-084.webp

* Update img-101.webp

* citation

* Update img-098.webp

* Update img-005.webp

* Update img-001.webp

* Update img-010.webp

* Update img-014.webp

* citation

* Update img-004.webp

* Update img-007.webp

* Update img-009.webp

* Update img-006.webp

* Update img-000.webp

LoicPandul committed:
ECO102 review (#1001)

* Update IMG02.webp

* "> L'État, c'est la grande fiction..." (img->txt)

* Update fr.md

* Delete IMG20.webp

* img bug

* french img

* Update IMG7.webp

* Laffer img

* img bug

* Update IMG4.webp

* Update fr.md

LoicPandul committed:
[TUTORIAL] Add Bitfinex (#991)

* txt

* Create cover.webp

* Create tutorial.yml

* img

LoicPandul committed:
Add additional "Conclusion" section (#1005)

midimarcus committed:
removal of passed exams in the events folder

trigger-xyz committed:
fix(bet): 21posters links

jramos0 committed:
new professors (#1029)

* New added teachers profile

* Bitcoin Baden Conference

LoicPandul committed:
Update tutorial logos (#1031)

* "muum" -> muun

* logo satochip

* ocean pool logo

* coinjoin dojo logo

* coinjoin SW logo

* coinjoin sparrow logo

* Labelling logo

* WST logo

* mnemonic logo

* github logo

* git logo

* IVPN logo

* mullvad logo

* veracrypt logo

* add bet logo

* add builder logo

* conference logo

* add book

* add podcast logo

* contribution correction logo

* new langage logo

* translate contribution logo

midimarcus committed:
initial tuto for b-cert verification

midimarcus committed:
fix ₿ and headers

midimarcus committed:
added Italian version

Asi0Flammeus committed:
Update en.md

Asi0Flammeus committed:
Update it.md

midimarcus committed:
adding Decoding Bitcoin course into event section (#1037)

* adding Decoding Bitcoin course into event section

* minor change to event description

* adding timezone data

LoicPandul committed:
links (#1036)

midimarcus committed:
added cover and logo images

Asi0Flammeus committed:
add batch translation

trigger-xyz committed:
Merge pull request #1032 from PlanB-Network/bcert-passed-exam-removal

removal of passed exams in the events folder

trigger-xyz committed:
Merge pull request #1035 from PlanB-Network/bcert-verification-tuto

initial tuto for b-cert verification

Asi0Flammeus committed:
btc204, update review chapter title

midimarcus committed:
Certificates timestamping tuto (#1039)

* updated cover and logo images

* changed tuto name and content to reflect general purpose of certificates verification

* removed old folder named after b-cert and old translations

* tuto, update verification

* tuto, add translation

* tuto, fix image name

* tuto, convert to webp

---------

Co-authored-by: Asi0Flammeus <asi0.flammeus@pm.me>

Asi0Flammeus committed:
tuto, fix image path

Asi0Flammeus committed:
tuto, fix image path

Asi0Flammeus committed:
btc204, fix typo chapter

jramos0 committed:
New thumbnail bringing Bitcoin & Spanish Version of Builders (#1041)

Asi0Flammeus committed:
tuto, fix plan ₿ symbol

Asi0Flammeus committed:
bet, add node nation

Asi0Flammeus committed:
Full review quizz content (#1048)

* CYP201 quizz (#975)

* Remove old quizz

* 2.1

* 2.2

* 2.3

* 3.1

* 3.2

* 3.3

* 4.1

* 4.2

* 4.3

* 4.4

* 5.1

* 5.2

* 5.3

* 5.4

* 6.1

* 6.2

* apostrophe yml

* [QUIZZ] PHI201 explanations keys (#1024)

* 119 -> 069

* 068 -> 000

* Update fr.yml

* Update fr.yml

* Update fr.yml

* Update fr.yml

* [QUIZZ] ECO102 (#1000)

* 1.1 -> 4.4

* quizz 5.1 -> 7.2

* apostrophe

* [QUIZZ] LNP201 (#1004)

* -> 007

* Update fr.yml

* I’m deleting everything and starting from scratch because the quiz is too flawed across all questions to correct.

* new 55 questions (5/chapter)

* apostrophe

* lnp201, temporary remove content

* lnp201, new fr version

* LNP201 complete overhaul (#987)

* delete old translations. keep only fr.md

* 1.1

* 1.2

* 2.1

* Update fr.md

* 2.2

* 2.3

* 2.4

* 3.1

* 3.2

* 3.3

* 4.1

* remove old img

* first 70 new img (until 4.1)

* 4.2

* Conclusion

* Update fr.md

* orthographe

* "Satoshis" -> "satoshis"

* 14 last img

* crop img

* img path in the text

* hours : 3

* intro

---------

Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

* lnp201, update image ref

* his201, reorder image ref

* [QUIZZ] CYP302 (#1007)

* 000 -> 024

* tags issue

* ... -> 034

* 035 - 059

* 060 -> 089

* 090 -> 114

* 115 -> 154

* apostrophe

* auto-translate, add automatic translation update for course folders

* autotranslate, quizz (part I)

* his201, fix it images

* [SCU101] Edit course and questions (#1030)

* course and questions review

* deleted questions 043 and 065

* scu101, prepare quizz

* Btc101 quiz review (#1042)

* update_BTC101_quizzes

* quizz_update_2

---------

Co-authored-by: Mari <Mari@jjhodl.com>
Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

* btc101, prepare quizz for translation

* quizz, add batch translation

* his201, fix images

* reviewed quizzes min201 (#1044)

* btc101, prepare quizz for translation

* quizz, fix explanation format

* min201, add quizz batch translation

* quizz, fix yml format for it

* quizz, fix yml format for cs

* quizz, fix yml format for de

* quizz, fix yml format for es and pt

* quizz, fix yml format for fi

* quizz, fix yml format for id

* quizz, fix yml format for vi

* quizz, fix yml format for zh-hans

* quizz, fix yml format for ru

* quizz, fix yml format for zh-hans

* quizz, fix yml format for et

* quizz, fix yml format for vi

* quizz, fix yml format for ja

* quizz, fix yml format for ru

* quizz, fix yml format for nb-NO

* quizz, fix yml format for fr

* quizz, fix yml format for pt

* quizz, fix yml format for et

* quizz, fix minor yml format

* quizz, fix minor yml format for zh-hans

* quizz, fix minor yml format for fi

* quizz, fix yml format for de

* quizz, fix yml format for wrong_answers

* quizz, fix yml format for wrong_answers

* quizz, fix yml format for fi

---------

Co-authored-by: Loïc Morel <137194052+LoicPandul@users.noreply.github.com>
Co-authored-by: Bitpreto <50926138+PaoloFoti@users.noreply.github.com>
Co-authored-by: MarJJ <mariannacon@protonmail.com>
Co-authored-by: Mari <Mari@jjhodl.com>

Asi0Flammeus committed:
courses, fix error format

Asi0Flammeus committed:
courses, fix error format

Asi0Flammeus committed:
courses, fix error format

Asi0Flammeus committed:
btc204, fix uuids

Asi0Flammeus committed:
courses, fix uuids and header format

Asi0Flammeus committed:
courses, fix uuids and header format

Asi0Flammeus committed:
courses, fix uuids and header format

Asi0Flammeus committed:
quizz, add missing indentation for explanation

Asi0Flammeus committed:
quizz, replace : by ;

Asi0Flammeus committed:
quizz, remove quote symbole

Asi0Flammeus committed:
script, remove fix script

Asi0Flammeus committed:
quizz, remove ``` strings

Asi0Flammeus committed:
quizz, remove '' quotes

Asi0Flammeus committed:
quizz, fix reviewed indentation

Asi0Flammeus committed:
quizz, fix reviewed for zh-hans

Asi0Flammeus committed:
cyp201, remove 1 wrong answer quizz

Asi0Flammeus committed:
quizz, fix yaml format

Asi0Flammeus committed:
quizz, fix reviewed for zh-hans

Asi0Flammeus committed:
quizz, fix reviewed for zh-hans

LoicPandul committed:
[TUTORIALS] Update titles (#1050)

* JADE -> Jade

* ZEUS -> Zeus

* Samouraï -> Samourai

* NunChuk -> Nunchuk

* HODL HODL -> Hodl Hodl

* MULLVAD VPN -> Mullvad VPN

* KEEPASS -> KeePass

* PROTON MAIL -> Proton Mail

* PROTON DRIVE -> Proton Drive

* YUBIKEY -> YubiKey

* AUTHY -> Authy

* BITWARDEN -> Bitwarden

Asi0Flammeus committed:
quizz, fix reviewed for zh-hans

Asi0Flammeus committed:
quizz, fix chapterId

Asi0Flammeus committed:
quizz, fix reviewed for zh-hans

rtbit committed:
Update nb-NO.md (#1049)

* Update nb-NO.md

* Update nb-NO.md

"wallets Bitcoin" is updated

LoicPandul committed:
[COURSES] BTC101 links (#1025)

* HIS201 + ECO102 links

* eco204 link

* CYP201 link

* only raw links

* only raw links 2

* raw links for preview

* passphrase*

* wallets tuto links

* btc102

* min201 link

* mercants tutos links

* lnp201 link

Commit:
update_proofreading metadata (#1045)

* update_proofreading metadata

* error_correction

---------

Co-authored-by: Mari <Mari@jjhodl.com>

alevandx committed:
[TUTORIAL] Add Elysium Wallet v1.0 (#978)

* add Elysium Wallet

* Update it.md

* Create extract1.webp

* Update it.md

* Delete en

* Update tutorial.yml

* imm update

* fix the rendering of images

---------

Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

Asi0Flammeus committed:
Cyp201 add translation (#1059)

* remove translations

* Delete loic_.webp

* remove img

* 2.1

* 2.2

* 2.3

* 3.1

* 3.2

* 3.3

* Update fr.md

* SIGHASH FLAGS new chapter

* 4.1

* 4.3

* 4.3

* 4.4

* 5.1

* 5.2

* 5.3

* 5.4

* 5.6 (descriptors)

* 5.5

* 5.7

* 5.7 last part + conclusion + intro + accroche

* img

* img path

* Update fr.md

* rognage

* proofreading

* hours: 4

* Update fr.md

* delete old quizz

* new quizz 90 questions

* apostrophe

* partie 1/2 proofreading

* fix latex

* suite

* Update fr.md

* Update fr.md

* Update fr.md

* cyp201, remove content to avoid conflict

* cyp201, add translation

---------

Co-authored-by: Loïc Morel <137194052+LoicPandul@users.noreply.github.com>

Asi0Flammeus committed:
cyp201, fix cs quizz yml format

Asi0Flammeus committed:
cyp201, fix quizz yml format

Asi0Flammeus committed:
cyp201, fix quizz yml format

Asi0Flammeus committed:
cyp201, fix quizz yml format

Asi0Flammeus committed:
cyp201, fix quizz yml format

Asi0Flammeus committed:
cyp201, add missing pieces to translation

Asi0Flammeus committed:
cyp201, fix quizz yml format

Asi0Flammeus committed:
btc102, fix duplicate proofreader id

Asi0Flammeus committed:
tuto, add missing id

Asi0Flammeus committed:
cyp201, fix quizz yml format

Asi0Flammeus committed:
cyp201, remove : from quizz

Asi0Flammeus committed:
cyp201, fix quizz yml format

Asi0Flammeus committed:
cyp201, fix quizz yml format

Asi0Flammeus committed:
cyp201, fix quizz yml format

Asi0Flammeus committed:
cyp201, fix quizz yml format

Asi0Flammeus committed:
cyp201, fix quizz yml format

Asi0Flammeus committed:
btc101, add conclusion part

Asi0Flammeus committed:
btc101, add conclusion part

Asi0Flammeus committed:
btc101, test header 5 for conclusion

Asi0Flammeus committed:
btc101, test header 5 for conclusion

Asi0Flammeus committed:
btc101, replace header 4 for conclusion

Asi0Flammeus committed:
courses, replace with Plan ₿

Asi0Flammeus committed:
btc102, add conclusion part

Asi0Flammeus committed:
eco102, add conclusion part

Asi0Flammeus committed:
eco104, add final part

Asi0Flammeus committed:
eco104, add uuid

Asi0Flammeus committed:
scu101, add final part

Asi0Flammeus committed:
btc204, update final part

Asi0Flammeus committed:
btc204, update final part

Asi0Flammeus committed:
cyp201, update final part

Asi0Flammeus committed:
eco201, add final part

Asi0Flammeus committed:
eco204, add final part

Asi0Flammeus committed:
his201, add final part

Asi0Flammeus committed:
lnp201, add final part

Asi0Flammeus committed:
min201, add final part

Asi0Flammeus committed:
phi201, add final part

BitPreto committed:
[MIN201] Text course review and images rename (#1079)

* initial review en text

* finished text review

* images updated across all lang files

* min201, fix typo in header

---------

Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

LoicPandul committed:
[TUTORIAL] Add Green Liquid (#1072)

* Create cover.webp

* Create tutorial.yml

* Create logo.webp

* txt

* orthographe

* img

* rognage

* Update 30.webp

* Update fr.md

* img path

LoicPandul committed:
[TUTORIAL] Add Green Desktop + HW (#1063)

* Create cover.webp

* Create logo.webp

* Create tutorial.yml

* txt

* orthographe

* img

* rognage

* Update fr.md

* img path

LoicPandul committed:
[TUTORIAL] Add Green Wallet watch-only (#1061)

* Create cover.webp

* Create logo.webp

* txt

* Create tutorial.yml

* Update fr.md

* img

* img path

* rognage

* Update fr.md

LoicPandul committed:
[TUTORIAL] Add Blockstream Green (v2) (#1055)

* logo blockstream -> logo green

* remove img

* add prof

* remove old tuto

* docs

* txt

* img

* rognage

* Update fr.md

* orthographe

* Update cover.webp

* img paths

LoicPandul committed:
Update tutorial.yml (#1082)

Asi0Flammeus committed:
bet, fix download url

LoicPandul committed:
Create tutorial-creation-guidelines.md (#1092)

LoicPandul committed:
[TUTORIAL] Add Blockstream Green multisig 2-of-2 "2FA" (#1083)

* Create tutorial.yml

* Create logo.webp

* Create cover.webp

* txt

* Update fr.md

* img

* rognage

* Update fr.md

LoicPandul committed:
Add python GUI for tutorial creation (#1095)

* Create new-tutorial-creation.py

* Update new-tutorial-creation.py

* .

* DARK MODE

* Update README.md

* Update README.md

LoicPandul committed:
[TUTORIAL] Add Aqua Wallet (#1111)

* Create cover.webp

* Create logo.webp

* Create fr.md

* Create tutorial.yml

* txt

* Update fr.md

* img

* Rognage

* Update 01.webp

* Update 04.webp

* Update 05.webp

* img paths

BitPreto committed:
add newsletters: bitcoin, IT ones (#1125)

* add newsletters: bitcoin, IT ones

* newsletter, add description directly in newsletter.yml

---------

Co-authored-by: Asi0Flammeus <asi0.flammeus@pm.me>

Asi0Flammeus committed:
newsletter, fix thumbnail

jramos0 committed:
New Builders and Tutorials (#1081)

* New Builders and Speed tutorial

* Update BTCPayServer tutorial

* Update BTCPayServer tutorial and Speed tutorial

* Braiins builder and mini miner tutorial

* fixed braiins mini miner tutorial

* fixed braiins mini miner tutorial mining

* taipei node builder

* PlanB Forum Lugano 2025 #1099

* Adopting bitcoin 2025,  #1102

* Adopting bitcoin cape town 2025, #1103

* vancouver learning bticoin 2025, #1105

* Btc prague 2025 and fixed thumbnails, #1106

* Bitcoin Las vegas #1112

* Add 4NK Academy, #1116

* Fix on Plan B Forum SV, #1097

* Bitcoin Alive #1104

* Bitblockboom Conf , #1107

* Tap Conf 2025 , #1124

* Bitcoin FIlm Fest #1109

* [conference] - Fix btc atlantis link 2024 #52

* BEA Events from Amity Age

Asi0Flammeus committed:
prof, update profile

Asi0Flammeus committed:
btc205, remove course because of language divergence

LoicPandul committed:
[COURSES] Update all courses durations (#1058)

* BTC 101

* BTC 102

* BTC 302

* BTC 305

* ECO 104

* ECO 201

* ECO 204

* HIS 201

* MIN 201

* SCU 101

* BTC101 - 7h

* BTC102 - 9h

* BTC204 - 26h

* BTC302 - 11h

* BTC305 - 15h

* CYP201 - 10h

* CYP302 - 16h

* ECO102 - 5h

* ECO104 - 8h

* ECO201 - 5h

* ECO204 - 6h

* HIS201 - 15h

* LNP201 - 6h

* MIN201 - 6h

* PHI201 - 11h

* SCU101 - 6h

BitPreto committed:
[MIN201] Fix image + add images .pptx (#1080)

* initial review en text

* finished text review

* images updated across all lang files

* min201, fix typo in header

* images.pptx created

* fix image 18.webp

* Cancelled ":" after Curriculum

* Wrong ":" n.2

---------

Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>
Co-authored-by: MarJJ <mariannacon@protonmail.com>

Commit:
[LNP201] adding_missing_images_to_all_languages (#1136)

* adding_missing_immages_to_all_languages

* adding_pptx

---------

Co-authored-by: Mari <Mari@jjhodl.com>

Commit:
Update README.md (#1135)

BitPreto committed:
Update 50-planb-tags.md (#1131)

small changes to better clarity of the document

BitPreto committed:
add btcita newsletter (#1130)

Commit:
BTC101 Typo fix and content review (#944)

* Typo fix

* Update en.md

* Update en.md

* Update en.md

* BTC101 update

* Update en.md

* Update en.md

* Update en.md

* Update en.md

* Update en.md

* Update en.md

* Update en.md

* Update en.md

* Final commit

* small addition

* Final correction

---------

Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

LoicPandul committed:
[TUTORIAL] Add Bacca (Ledger Installer) (#1138)

* Create fr.md

* Create tutorial.yml

* Create logo.webp

* Create cover.webp

* Real name -> Bacca

* Delete fr.md

* txt

* Update fr.md

* img

* rognage

* img paths

* Update fr.md

LoicPandul committed:
[COURSE] CYP201 correction (#1134)

* "Plan ₿" -> "PlanB"

We can’t change the ones in the examples for hash functions otherwise the hash in examples won’t be correct.

* Question 028 correction

2 good responses ^^

* Plan ₿ broken link

* LATEX correction

* Update cs.md

* Update cs.md

* Update fr.md

* Update en.md

* Update en.md

* Update de.md

* Update de.md

* Update fr.md

* Update es.md

* Update de.md

* Update et.md

* Update fi.md

* Update id.md

* Update it.md

* Update ja.md

* Update nb-NO.md

* Update pt.md

* Update ru.md

* Update vi.md

* Update zh-Hans.md

* Update et.md

* Update fi.md

* Update id.md

* Update ja.md

* Update vi.md

* Update id.md

* Update it.md

* Update nb-NO.md

* Update pt.md

* Update ru.md

* Update zh-Hans.md

LoicPandul committed:
[TUTORIAL] Add Alby Hub (#1126)

* Create tutorial.yml

* Create cover.webp

* Create fr.md

* cat: wallet -> LN node

* txt

* Update fr.md

* Update fr.md

* Update fr.md

* img

* Update fr.md

* rognage

* img path

LoicPandul committed:
Create update-all-courses-hours.py (#1085)

BitPreto committed:
[ECO102] Course review EN,FR text typos and images (#1093)

* eco102, update review

* image structure

* reviewed text+images en,fr .md files

* remove quizzes from current edits to be merged

* edit last proofread date

* update branch with up-to-date quizzes from dev

---------

Co-authored-by: Asi0Flammeus <asi0.flammeus@pm.me>

Asi0Flammeus committed:
events, fix typos

Asi0Flammeus committed:
newsletters, test new link

Asi0Flammeus committed:
newsletters, fix link

LoicPandul committed:
URL fix tuto links (#1160)

* fix hypertext redirection for tutos

* fix hypertext redirection for courses

Asi0Flammeus committed:
newsletters, fix link

BitPreto committed:
[RESOURCE] Add Newsletters (#1152)

* en,it newsletters text

* en,it newsletters images

* Changes for better clarity

* Update newsletter.yml

* Description correction

* Corrections for more fluidity

* added en,it,fr newsletters

* description fix

* description update

* Descritpion update 2

* Update newsletter.yml

* Update newsletter.yml

* Update newsletter.yml

* Update newsletter.yml

* Update

* Update

* Update newsletter.yml

* add en,it,fr newsletters images

* edit news image previous batch

---------

Co-authored-by: MarJJ <mariannacon@protonmail.com>

jramos0 committed:
New Builders for validation and Events (#1167)

* [Fix] Update Builders and fix Validations

* Fix on TABConf thumbnail #1124

* BTC Helsinki #1110

* Understanding Bitcoin Roatan Conference #1186

* Bitcoin Medellin #1184

* bitcoinday naples #1185

* Mining Disrupt 2025 #1187

* Cheatcode benford #1188

* The Bitcoin Rodeo #1191

* Watch Out Bitcoin #1189

* Bitcare Italy #1190

* Meetup Belgrade Dec-25

LoicPandul committed:
Update the `builder:` keys in the tutorials (#1192)

* Stonewall x2

* bip47

* wst

* labbelling

* payjoin sparrow

* coinjoin sparrow

* analysis oxt

* boltzmann

* payjoin

* joinbot

* ricochet

* coinjoin SW

* payjoin SW

* coinjoin dojo

* remix whirlpool

* stonewall

* Add Nunchuk builder (translation done)

* bitkit

* mnemonic

* recovery test

* samourai recovery

* satscard

* sentinel

* tapsigner

* add Satochip builder (translation done)

* passphrase

* add book

* add builder Proton (translation done)

* tuto add builder

* add new langage tuto

* correct error tuto

* github dekstop

* Add builder Tor project (Translation done)

* add bet tuto

* add conference tuto

* add event tuto

* add podcast tuto

* content review tuto

* create teacher tuto

* add builder GrapheneOS (translation done)

* security key

* Create logo.webp

* logo tor webp

* Linux mint

* weblate tuto

* Add GitHub Builder (translation done)

* github basics

* github desktop

* Add builder Mullvad (translation done)

* integrity authenticity

* add builder keepass (translation done)

* add builder Bitwarden (translation done)

* write tutorial

* add builder Veracrypt (translation done)

* add builder Twilio - Authy (translation done)

* add builder Public Pool (translation done)

* nerd miner

* add builder Nerd Miner (translation done)

* bitaxe

* add builder bitaxe (translation done)

* add translations on braiins builder

* braiins

* add Ocean pool builder (translation done)

* neutrino

* ronin

* add Bitcoin Core builder (translation done)

* Core linux

* Core windows

* lapp

* ronin v2

* green 2FA

* green desktop

* green watch-only

* green liquid

* green mobile

* add builder JAN3 (translation done)

* add builder ElysiumLab (translation done)

Asi0Flammeus committed:
initial upload of YT channel resources (#1216)

* channels, add initial upload

* channel, update thumbnail

Commit:
BTC101 - minor errors update and poster removal (#1193)

* Update of some minor errors for BTC101 text

* Update_2

* poster_delete

---------

Co-authored-by: Mari <Mari@jjhodl.com>

BitPreto committed:
[RESOURCE] Newsletters, fix images size and resolution (#1194)

* newsletters, fix images size and resolution, where needed

* add-21-es-newsletter

LoicPandul committed:
[TUTORIAL] Add Coldcard Q - Setup and basic usage (#1197)

* Create cover.webp

* Create logo.webp

* Create fr.md

* Create tutorial.yml

* Update fr.md

* Update tutorial.yml

* txt + relecture orthographe

* Update fr.md

* img

* rognage

* Update fr.md

* img paths

LoicPandul committed:
Update new-tutorial-creation.py (#1198)

LoicPandul committed:
Create logo.webp (#1199)

jramos0 committed:
[Conference Replay] Conference from 2024 (#1211)

* Conference Replay BTCPrague, closes #1201

* Conference Replay Adopting Bitcoin SV, closes #1202

* baltic honeybadger conference, closes #1203

* Nashville Bitcoin 2024 Conference replay , closes #1204

* WoBitcoin 2024, closes #1205

* Bitcoin Amsterdam 2024 , closes #1206

* Canadian Conference, closes #1208

* Plan B Forum Lugano 2024, closes #1209

* TABConf 6 , closes #1210

* Fix typo conference replay

* Le Paradigme Bitcoin , closes #1207

* Update en.md

* Update WoB24

---------

Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

LoicPandul committed:
[TUTORIAL] Add COLDCARD Q advanced options (#1212)

* Create cover.webp

* Create logo.webp

* Create fr.md

* Create tutorial.yml

* Add the COLDCARD logo to the old MK3 tutorial too

* txt

* Update fr.md

* Update fr.md

* img

* rognage

* img paths

LoicPandul committed:
Fix BTC204 link (#1213)

* Update fr.md

* Update cs.md

* Update de.md

* Update en.md

* Update es.md

* Update et.md

* Update fi.md

* Update id.md

* Update it.md

* Update ja.md

* Update nb-NO.md

* Update pt.md

* Update ru.md

* Update vi.md

* Update zh-Hans.md

LoicPandul committed:
Update fr.md (#1214)

Asi0Flammeus committed:
events, remove exclusive PlanB events

Asi0Flammeus committed:
professors, remove PlanB exclusive lecturers

zeevick10 committed:
Update en.md (#1215)

Pivii committed:
[TUTORIAL] New Lnp2pbot Tutorial (#1145)

* add new folder for the new version

* buy order images added

* update fr.md

* resize iamges

* add images and update fr.md

* add logo

* update fr.md

* add .yml

* update

* Update tutorial.yml

* update

* update fr.md

* update fr.md

* add link tuto wallets

* add telegram link

* refactor: update heading hierarchy levels throughout document

* builder and tags modification

---------

Co-authored-by: Pierre <pierre.vvr@protonmail.com>

Commit:
adding_missing_images (#1217)

Co-authored-by: Mari <Mari@jjhodl.com>

Commit:
CYP201 missing images check (#1218)

* adding_missing_images

* CYP201_missing_images_added

---------

Co-authored-by: Mari <Mari@jjhodl.com>

Commit:
Minor corrections (#1219)

jramos0 committed:
Conference Replay (#1227)

* BTC Asia, closes #1220

* Arnhem 2024 AB, closes #1221

* Bitcare Forum 2024 , closes #1222

* BTC Thailand conference 24, #closes #1223

* Swiss conference , closes #1124

* FBC 2024, closes #1225

* Builders for conference validation

* Bitcare forum builder

* Conference Builders: Adopting Bitcoin & Bitcoin Tokyo

* Fix on BTC Hel Event

* Fix on Bitcoin Tokyo Builder

* Surfin bitcoin 2024, closes #1242

* Bitcoin tokyo conference replay, #closes #1243

* [Event]Bictoin expo 25

* [Event] fix Bictoin expo 25

* Bitcoin++ Builder

* Conference Replays from Bitcoin++ , closes #1261 #1262 #1263

* Event OsloFF 25, closes #1264

* Blockchain Mallorca 2025

LoicPandul committed:
[BET] Add mnemonic sheet (#1228)

* Create bet.yml

* Create fr.yml

* Create cs.yml

* Create de.yml

* Create et.yml

* Create es.yml

* Create zh-Hans.yml

* Create en.yml

* Create fi.yml

* Create id.yml

* Create it.yml

* Create ja.yml

* Create nb-NO.yml

* Create pt.yml

* Create ru.yml

* Create vi.yml

* Create logo.webp

* Create mnemonic-sheet.pdf

LoicPandul committed:
[TUTORIAL] Add Backup Mnemonic (#1230)

* Create cover.webp

* Create logo.webp

* Create tutorial.yml

* txt

* Update fr.md

* img

* Update fr.md

* rognage

* Update fr.md

* img paths

* Update fr.md

LoicPandul committed:
Fix all tutorial links (#1235)

* fix link tuto green desktop

* fix link tuto blockstream liquid

* cs Recovery test tuto fix bloc code

* de Recovery test tuto fix bloc code

* en Recovery test tuto fix bloc code

* es Recovery test tuto fix bloc code

* et Recovery test tuto fix bloc code

* fi Recovery test tuto fix bloc code

* id Recovery test tuto fix bloc code

* it Recovery test tuto fix bloc code

* ja Recovery test tuto fix bloc code

* pt Recovery test tuto fix bloc code

* ru Recovery test tuto fix bloc code

* vi Recovery test tuto fix bloc code

* zh-Hans Recovery test tuto fix bloc code

* green mobile link in BTC102

* BTC204 link tuto coinjoin

* BTC204 link tuto payjoin

* all BTC204 links to tuto

* SEC101 link to keypass tuto

* ronin dojo tuto links

* create teacher tuto link

* Coinjoin dojo tuto links

* coinjoin samourai wallet tuto links

* coinjoin sparrow wallet tuto link

* oxt analysis tuto links

* codlcard q tuto links

* SW recover tuto link

* sentinel tuto link

* sentinel tuto link

* recover SW tuto links

* oxt analysis tuto links

* tuto coinjoin SW links

* sparrow wallet coinjoin tuto links

* coinjoin dojo tuto links

* oxt analysis tuto links

* oxt analysis tuto links

* oxt analysis tuto links

* BTC204 links

* Update README.md

* Update tutorial-creation-guidelines.md

* Delete broken_links_report.md

* links preview

* add Bitrefill tuto link BTC204

LoicPandul committed:
add link to COLDCARD advanced tuto (#1237)

LoicPandul committed:
[COURSE] BTC204 - Add FR videos (#1241)

* Update fr.md

* nouvelle intro

Commit:
Link correction and disclaimer (#1260)

toofooboo committed:
chore: fix some typos in README.md (#1265)

Signed-off-by: toofooboo <cmaker@foxmail.com>

LoicPandul committed:
[TUTORIAL] Edit - review contribution (#1269)

* Update fr.md

* Update fr.md

* new img

* remove old img

* Update en.md

* update

* Update fr.md

* Update en.md

* delete old translations

* Fix Plan ₿ Network

---------

Co-authored-by: MarJJ <mariannacon@protonmail.com>

BitPreto committed:
[RESOURCE] Add YT channels, Edit Newsletters (#1315)

* add 1st batch of yt channels

* fix newsletters ambiguity

* add 2nd batch of yt channels, multiple langs

* new batch of podcasts

* add 3rd batch of yt channels

Asi0Flammeus committed:
channel, fix indentation

LoicPandul committed:
Update btc204 fr (#1340)

LoicPandul committed:
remove v1 tutorial (#1342)

Reshetenko committed:
Robosats_Hodlhodl_Bisq - Russian proofreading (#1195)

* Robosats

Robosats tutorial

* HodlHodl

HodlHodl tutorial

* Bisq

Bisq Tutorial

yutaro21jp committed:
Update ja.md (#1144)

Done! : )

yutaro21jp committed:
Update ja.md (#1143)

Done! : )

yutaro21jp committed:
Lapp Japanese proofreading (#1142)

* Update ja.md

Done! : )

* Update ja.md

---------

Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

yutaro21jp committed:
Update ja.md (#1132)

Done! : )

yutaro21jp committed:
Update ja.md (#1129)

Done! : )

Reshetenko committed:
Muun and Blixt tutorial Ru proofreading (#1127)

* All tutorial

All tutorial

* Blixt

Blixt tutorial

yutaro21jp committed:
Update ja.md (#1115)

Done! : )

yutaro21jp committed:
Update ja.md (#1114)

Done! : )

yutaro21jp committed:
Update ja.md (#1113)

Done! : )

yutaro21jp committed:
Update ja.md (#1090)

Done! : )

yutaro21jp committed:
Update ja.md (#1089)

Done! : )

yutaro21jp committed:
Update ja.md (#1084)

Done! : )

yutaro21jp committed:
Update ja.md (#1078)

Done! : )

yutaro21jp committed:
Update ja.md (#1076)

Done! : )

yutaro21jp committed:
Update ja.md (#1075)

Done! : )

yutaro21jp committed:
Update ja.md (#1074)

Done! : )

yutaro21jp committed:
Update ja.md (#1073)

Done! : )

yutaro21jp committed:
Update ja.md (#1071)

Done! : )

yutaro21jp committed:
Update ja.md (#1070)

Done! : )

yutaro21jp committed:
Update ja.md (#1069)

Done! : )

yutaro21jp committed:
Update ja.md (#1068)

Done! : )

yutaro21jp committed:
Update ja.md (#1066)

Done! : )

yutaro21jp committed:
Update ja.md (#1065)

Done! : )

yutaro21jp committed:
Update ja.md (#1064)

Done! : )

yutaro21jp committed:
Update ja.md (#1057)

Done! : )

yutaro21jp committed:
Update ja.md (#1056)

Done!

Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

yutaro21jp committed:
Update ja.md (#1054)

Done!

LoicPandul committed:
[TUTORIAL] Add tuto Bull Bitcoin Europe (new platform) (#1348)

* Create cover.webp

* Create fr.md

* Create tutorial.yml

* txt

* Update fr.md

* img

* rognage

* Update fr.md

* img paths

LoicPandul committed:
[COURSES] Fix CYP201 bullet list (#1349)

* Update cs.md

* Update de.md

* Update en.md

* Update es.md

* Update et.md

* Update pt.md

* Update fi.md

* Update zh-Hans.md

* Update ru.md

* Update nb-NO.md

* Update ja.md

* Update it.md

* Update id.md

* Update fr.md

* Update vi.md

LoicPandul committed:
[TUTORIAL] Fix LNP2PBOT v2 (#1350)

* Update fr.md

* Update fr.md

LoicPandul committed:
[COURSES] Fix quizz difficulty ECO201 and MIN201 (#1351)

* Update question.yml

* ECO201

Asi0Flammeus committed:
update node nation bet

Asi0Flammeus committed:
Update translation script processes (#1357)

* auto-translate, testing new worflows

* auto-translate, new json format

* add test json

* auto-translate, add original content in json

* auto-translate, add course cost analysis

* update reverse conversion and test

* rename folder auto-translate

* make translation controller

* fix EN for deepl as source language

* add properties from question.yml

* fix supported language dict

* adapte auto-update for tutorial with new translation process

* add missing translations for tutorials

* update gitignore

* remove old structure for translation

* fix auto-translate for courses

* clean auto-translate folders

* fix books

* add auto-translate for ressource (attempt)

* update object detector

* adapt json conversion process to support yml files

* support yml file conversion and translation

* book, fix use of : in description

* add helper for adding new supported languages

* add auto translation for initial upload

* auto-translate, add skip folders for initial translation

Asi0Flammeus committed:
script, add professor path for auto-translate

LoicPandul committed:
[TUTORIALS] Correct tutorials name (#1359)

* name:

* name:

LoicPandul committed:
Add a paragraph on authentication + fix links (#1361)

LoicPandul committed:
fix URL links (#1362)

LoicPandul committed:
remove youtube link (#1367)

LoicPandul committed:
[TUTORIAL] Add Relai v2 (#1369)

* remove old tuto

* Create cover.webp

* Create 01.webp

* Create 02.webp

* Create 03.webp

* Create 04.webp

* Create tutorial.yml

* txt

* img paths

* fr -> en

* Create profile.webp

* Create professor.yml

* Update tutorial.yml

* prof descriptions (all languages)

* relai -> relaiv2

BitPreto committed:
1 Resource (fix) and many tutos edit (#1365)

* edit pbn tags

* typo fixes for newsletters' tags

* tutos, fix typos

* tutos, 2nd fix typos

* tutos, 3rd fix typos

* book, fix author zero to one

* tutos, batch1 edits

* tutos, batch2 edits

BitPreto committed:
[RESOURCE] Add Builders, Fix Conferences (#1376)

* new builders

* fix conferences and a tutorial

networkeffect committed:
Biz101 (#1238)

* premier brouillon en

* brouillon

* brouillon

* brouillon

* brouillon

* brouillon

* brouillon

* brouillon

* brouillon final

* Delete custodym.webp

* Create custodym.webp

* review

* question yaml

* difficulty quizz

* quizz tags

* Update course.yml

* Update professor.yml

* add translations prof description

* Update 01.webp

* Update 03.webp

* Update 04.webp

* Delete 06.webp

* Update 07.webp

* Delete 08.webp

* Update 09.webp

* Delete 10.webp

* Delete 11.webp

* Update 12.webp

* Delete 13.webp

* Update 14.webp

* Delete 15.webp

* Delete 16.webp

* Delete 17.webp

* Delete 18.webp

* Update 19.webp

* Update en.md

* Create en.yml

* Create question.yml

* Create en.yml

* Create question.yml

* Create en.yml

* Create question.yml

* Update 04.webp

* Update en.md

* update gitignore

* biz101, update image name and add thumbnail

---------

Co-authored-by: Loïc Morel <137194052+LoicPandul@users.noreply.github.com>
Co-authored-by: Asi0Flammeus <asi0.flammeus@pm.me>
Co-authored-by: asi0 <90136504+Asi0Flammeus@users.noreply.github.com>

Asi0Flammeus committed:
Update translation for new tutorials (#1377)

* tutorials, add missing translation

* scripts, add professors for initial uplaod translation

* auto-translate, add requirements

* batch translate, update tutorials

---------

Co-authored-by: Asi0Flammeus <datascript@ns3104763.ip-145-239-142.eu>

Asi0Flammeus committed:
courses, fix review chapter name

Asi0Flammeus committed:
courses, fix conclusion chapter

jramos0 committed:
[TESTNET] Liquid Courses drafts (#1356)

* Liquid Elements course draft

* Liquid Elements course draft1.1

* Liquid Elements course draft1.2

* Liquid Elements course draft1.3

* Liquid Elements course draft1.4

* Liquid Bootcamp draft

* Liquid Bootcamp draft 1.2

* Liquid Bootcamp draft 1.3

* Liquid Bootcamp draft 1.4

* Liquid Bootcamp draft 1.5

* Liquid Bootcamp draft 1.5

* adapt indexes

* sid202, format codeblock

* change level

* remove final exam chapter

* remove final exam chapter

* sid302, add thumbnail

---------

Co-authored-by: Asi0Flammeus <asi0.flammeus@pm.me>

Asi0Flammeus committed:
fix builder website

Asi0Flammeus committed:
biz101, fix quizz indentation

Asi0Flammeus committed:
biz101, fix uuids

Asi0Flammeus committed:
sid202, update thumbnail

Asi0Flammeus committed:
courses, remove content not ready for mainnet deployment

trigger-xyz committed:
fix merge issue

trigger-xyz committed:
Merge pull request #1378 from PlanB-Network/fix-final-chapters

Fix final chapters

Asi0Flammeus committed:
add back pending courses

Asi0Flammeus committed:
btc101, fix typo

Asi0Flammeus committed:
builder, fix category upper case

DavidSeas committed:
Update it.md (#989)

Fix translation error

Asi0Flammeus committed:
Update tutorial.yml

Asi0Flammeus committed:
replace by peertube video

Asi0Flammeus committed:
auto-translate, add deepseek

Asi0Flammeus committed:
auto-translate, fix indent

Asi0Flammeus committed:
zeus, update es version

Asi0Flammeus committed:
update gitignore

Asi0Flammeus committed:
Merge branch 'dev' of github.com:Darth-Coin/sovereign-university-data into Darth-Coin-dev

Asi0Flammeus committed:
zeus, remove mp4 files

Asi0Flammeus commented:
I have fixed some minor things: 
- remove links in credit because not supported
- switch mp4 file by self-hosted peertube videos

Now it can be merged, and it will be soon translated in all supported languages. 
Thanks for your contribution!
